packages/components/src/components/SquiggleViewer/VariableBox.tsx

```diff
@@ -103,13 +106,9 @@ export const VariableBox: FC<VariableBoxProps> = ({
     };
   };
 
-  const setSettingsAndUpdate = (newSettings: LocalItemSettings) => {
-    setSettings(path, newSettings);
-    forceUpdate();
-  };
-
   const toggleCollapsed = () => {
-    setSettingsAndUpdate({ ...settings, collapsed: !settings.collapsed });
+    toggleCollapsed_(path);
```

Comment:
@berekuk : Was there a good reason why we didn't use a toggleCollapsed dispatch method for this? Seems much cleaner.